restore/restore.go

```diff
@@ -125,17 +125,38 @@ func getRemoteBackup(r *Restore, conf *config.Config) {
 		log.Fatalf("[ERR] Unable to create local restore temp file!: %v", err)
 	}
 
-	// Create the params to pass into the actual downloader
-	params := &s3.GetObjectInput{
-		Bucket: &conf.S3Bucket,
-		Key:    &r.RestorePath,
-	}
+	if len(string(conf.GCSBucket)) < 1 {
```

Comment:
Similar to the above comments, can this be moved into its own dedicated function, like `getRemoteBackupS3()` and `getRemoteBackupGoogleStorage()` ?